src/routes/+page.svelte

```diff
@@ -1,77 +1,62 @@
 <script lang="ts">
-	import StoragePlace from './StoragePlace.svelte';
-	import {
-		IconArrowUp,
-		IconArrowDown,
-		IconBackspace,
-		IconArrowBack,
-		IconArrowRight,
-		IconArrowLeft
-	} from '@tabler/icons-svelte';
+  import StoragePlace from '$lib/components/StoragePlace.svelte'
 
-	let showTips = $state(false);
-	// let tips: HTMLDivElement | null = $state(null);
-	let tips: HTMLDivElement;
+  let showTips = $state(false)
+  let tips: HTMLDivElement | null = $state(null)
 
-	// TODO: fix broken accordion on resize
+  $effect(() => {
```

Comment:
just using `let tips: HTMLDivElement` gave a warning so to use `$state` on an element, we modify it in an `$effect` (and not in the template).
here, `$effect` acts as the old `onMount`, so runs once during component initialization.